templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
+companies do. At a company, users are on their company's chat all day. In a
+part-time working group or community, users might check the chat less than
+once a week. In such a group, a good chat system can be the difference
+between robust community and non-existence.
+
+Some of Zulip's earliest users were standards bodies and alumni groups, so
+we have given a lot of thought to the problems such groups face. Zulip's
+topic-based threading
+* Makes the catching-up experience fast and fun, so that users are
+  encouraged to come back. On Slack or email, wading through hundreds or
+  thousands of unread messages when you come back after a month is taxing at
+  best.
+* Makes it easy to respond to conversations that started hours or days ago,
+  so that users that drop by occasionally can contribute rather than just
+  lurk.
+
+Zulip's topic-based threading also allows for more thoughtful discussion,
+since more people are able to chime in on any given conversation.
+
+If you haven't read [why Zulip](/why-zulip), read that first. One of our
+earliest communities, the Recurse Center, also wrote an
+[extended blog post](https://www.recurse.com/blog/112-how-rc-uses-zulip)
+about how they use Zulip.
```

Comment:
maybe mention that in particular, that article has some great advice for conventions you might want to include to help build your community.